### DISCUSSION TITLE
How to start contributing to Robolectric

### DISCUSSION THREAD

**Discussion by Subhadiptech** [robolectric/robolectric#7908] [Q&A]
As I am new to this organisation I want help with some doc so that I can setup my project on local system.

**Comment by RohanMishra315** [comment]
@Subhadiptech hey could you help me how do i copy android dependencies on local maven repo with this ./scripts/install-dependencies.rb where to find local maven repo?? pls help

**Comment by utzcoz** [comment]
@Subhadiptech You can check https://robolectric.org/getting-started/ to setup Robolectric for your project.

**Comment by Subhadiptech** [comment]
@utzcoz actually I want to start contributing to Robolectric
